math_series/series.py

```diff
@@ -0,0 +1,39 @@
+def fibonacci(n):   
+    if n == 0: 
+        return 0
+    elif n == 1 or n == 2: 
+        return 1
+    else:
+        return n * fibonacci((n - 1 ) + (n - 2))
+
+def lucas(n):
+    if n == 1:
+        return 2
+    elif n == 2:
+        return 1
+    else:
+        return n * lucas((n - 1) + (n - 2))
+
+
+def sum_series(n, f_num=0, s_num=1):
+    if f_num == 0 and s_num == 1:
+        return fibonacci(n)
+    elif f_num == 2 and s_num == 1:
+        return lucas(n)
```

Comment:
While I definitely see the spirit of this solution, the sum_series function should work for any values of `f_num` and `s_num` here, not just the ones associated with fibonacci and lucas.